zip-0227.rst

```diff
@@ -337,13 +312,18 @@ If all of the above checks pass, do the following:
 - If :math:`\mathsf{finalize} = 1`, add :math:`\mathsf{AssetDigest}` to the :math:`\mathsf{previously\_finalized}` set immediately after the block in which this transaction occurs.
 - (Replay Protection) If issue bundle is present, the fees MUST be greater than zero.
 
+Rationale for non-zero Issue Bundle fees
```

Comment:
Rationale for non-zero fees when an issue bundle is included.

Reply:
Done